tests/image/image.cpp

```diff
@@ -1451,6 +1454,37 @@ void ImageTestCase::ScaleCompare()
                                "image/cross_nearest_neighb_256x256.png");
 }
 
+const unsigned char toucan[] =
+{
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x40, 0x00, 0x00, 0x01, 0xc0, 0x00,
+    0x00, 0x09, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00, 0x00, 0x34, 0x00, 0x00, 0x00, 0x34, 0x00, 0x00,
+    0x00, 0x78, 0x00, 0x00, 0x00, 0x78, 0x00, 0x00, 0x08, 0xf8, 0x00, 0x00, 0x58, 0xf8, 0x00, 0x00,
+    0x2f, 0xf8, 0x00, 0x00, 0x50, 0x78, 0x00, 0x00, 0x64, 0x00, 0x01, 0x00, 0x61, 0x80, 0x00, 0x00,
+    0x7b, 0x00, 0x00, 0x00, 0x00, 0x60, 0x00, 0x00, 0x28, 0x00, 0x00, 0x00, 0x0d, 0x20, 0x00, 0x00,
+    0x48, 0x10, 0x00, 0x00, 0x50, 0x80, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+};
+
+void ImageTestCase::CreateBitmapFromCursor()
+{
+#if !defined __WXOSX_IPHONE__ && !defined __WXDFB__ && !defined __WXMOTIF__ && !defined __WXX11__
+#ifdef __WXMSW__
+    wxImage image( 32, 32, toucan );
+#else
+    wxImage image( "../../samples/image/toucan.png" );
```

Comment:
In Travis, the tests are run from the `tests` directory, so I think you need only one `..`.
But when configure is not run in the main wx directory, and the test executable is created somewhere else, this will never find the image.

I think you need to add the image somewhere to `test.bkl` so the image will be copied to the runtime directory. Probably similar to the images in `data-images`. Then you can just use `image/toucan.png`

Reply:
Why do we use XPM under MSW and PNG file elsewhere? You could use easily `wxBITMAP_PNG_FROM_DATA` everywhere, probably with a small (16x16 or 32x32) subset of `toucan.png`.